docs/data-ai/capture-data/advanced/advanced-data-capture-sync.md

```diff
@@ -0,0 +1,626 @@
+---
+linkTitle: "Advanced data capture and sync configurations"
+title: "Advanced data capture and sync configurations"
+tags: ["data management", "data", "services"]
+weight: 10
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "Advanced data capture and data sync configurations."
+prev: /data-ai/capture-data/conditional-sync/
+date: "2025-02-10"
+---
+
+You can configure both the data management and data capture with attributes in JSON for advanced use cases.
+You can also configure data capture for remote parts.
+
+### Advanced data management service configurations
+
+To configure the data manager in JSON, see the following example configurations:
```

Comment:
```suggestion
To configure the data manager in JSON, see the following example configurations.
The attributes that are only available with advanced (JSON) configuration are highlighted below.
```
(If you choose to implement the comment below)

Reply:
Too likely to go out of date